rest/tests/test_rest_authorization.py

```diff
@@ -750,19 +754,30 @@ def test_resource_put_patch(self):
 
     def test_static_assets_get(self):
         """Test for getting static assets from learning_resources"""
-        resource1 = self.import_course_tarball(self.repo)
+        def get_resource_with_asset(type_name):
+            """
+            Get a LearningResource with a StaticAsset.
+            """
+            for resource in get_resources(self.repo.id):
+                if resource.learning_resource_type.name != type_name:
+                    continue
+                if resource.static_assets.count() == 0:
+                    continue
```

Comment:
you can have an `if` on multiple lines (using parenthesis)
I agree the functionality is the same, but two `if` statements with the same behavior should be merged